tiled/server/dependencies.py

```diff
@@ -163,7 +166,7 @@ def expected_shape(
 
 
 def slice_(
-    slice: Optional[str] = None,
+    slice: str = Query(None, pattern=SLICE_REGEX),
```

Comment:
Optional here indicates to OAS3 that `None` is a valid value.

```suggestion
slice: Optional[str] = Query(None, pattern=SLICE_REGEX),
```